articles/virtual-machines/linux/create-upload-centos.md

```diff
@@ -320,29 +320,29 @@ Preparing a CentOS 7 virtual machine for Azure is similar to CentOS 6, however t
    ```
     
    > [!Note]
-   > The rest of this guide will assume you're using at least the `[openlogic]` repo, which will be used to install the Azure Linux agent below.
+   > The rest of this guide will assume you are using at least the `[openlogic]` repo, which will be used to install the Azure Linux agent below.
 
 7. Run the following command to clear the current yum metadata and install any updates:
 
     ```bash
     sudo yum clean all
     ```
 
-    Unless you're creating an image for an older version of CentOS, it's recommended to update all the packages to the latest:
+    Unless you're creating an image for an older version of CentOS, it is recommended to update all the packages to the latest:
 
     ```bash
     sudo yum -y update
     ```
 
-    A reboot maybe required after running this command.
+    A reboot may be required after running this command.
 
 8. Modify the kernel boot line in your grub configuration to include additional kernel parameters for Azure. To do this, open `/etc/default/grub` in a text editor and edit the `GRUB_CMDLINE_LINUX` parameter, for example:
 
     ```config
     GRUB_CMDLINE_LINUX="rootdelay=300 console=ttyS0 earlyprintk=ttyS0 net.ifnames=0"
     ```
 
-   This will also ensure all console messages are sent to the first serial port, which can assist Azure support with debugging issues. It also turns off the new CentOS 7 naming conventions for NICs. In addition to the above, it's recommended to *remove* the following parameters:
+   This will also ensure all console messages are sent to the first serial port, which can assist Azure support with debugging issues. It also turns off the new CentOS 7 naming conventions for NICs. In addition to the above, it is recommended to *remove* the following parameters:

```

Comment:
```suggestion
   This will also ensure all console messages are sent to the first serial port, which can assist Azure support with debugging issues. It also turns off the new CentOS 7 naming conventions for NICs. In addition to the above, we recommend to *remove* the following parameters:
```